go.toolchain.rev

```diff
@@ -1 +1 @@
-16af3c2f4711e2bb6f662b2bc8d5cc6eb4c572cc
+d149af282305d5365d5a4fb576d9fa81247eb6da
```

Comment:
Entire diff is the VERSION file: https://github.com/tailscale/go/compare/16af3c2f4711e2bb6f662b2bc8d5cc6eb4c572cc..d149af282305d5365d5a4fb576d9fa81247eb6da